src/utils/withFetch.js

```diff
@@ -16,15 +16,16 @@ const withFetch = ( url, options = {}, name = null ) => {
 
 				// eslint-disable-next-line
 				this.state = {
-					loading: true,
-					expires: 0,
-					data:    {},
-					error:   false,
+					loading:  true,  // Initial load indicator.
+					fetching: false, // Refetch indicator.
+					expires:  0,
+					data:     {},
+					error:    false,
 				};
```

Comment:
Something I missed before, need to add `this.doFetch = this.doFetch.bind(this);` inside the constructor